cypress/TestCases/FireboltCertification/Manage/SecureStorageManage.feature

```diff
@@ -124,3 +124,81 @@ Feature: SecureStorage_Manage
             | Passing scope as null         | clear secure data values for an app with null scope    |
             | Passing scope as boolean      | clear secure data values for an app with boolean scope |
             | without scope                 | clear secure data values for an app without scope      |
+
+    @SecureStorage @manageSDK
+    Scenario Outline: SecureStorage.setForApp - Positive Scenario: <Scenario>
+        Given 3rd party 'certification' app is launched with 'secondary 3rd party app' appId
+        And 1st party app invokes the 'Firebolt' API to '<Clear_API_Key>'
+        And 'Firebolt' platform responds to '1st party app' for 'null for clearing stored value for an app'
+        When '3rd party app' invokes the 'Firebolt' API to '<API_Key>'
+        Then 'Firebolt' platform responds with 'null for getting stored value'
+        Given 'secondary 3rd party app' invokes the 'Firebolt' API to '<API_Key>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with 'null for getting stored value'
+        When 1st party app invokes the 'Firebolt' API to '<Set_API_Key>'
+        Then 'Firebolt' platform responds to '1st party app' for '<Set_API_Key>'
+        When '3rd party app' invokes the 'Firebolt' API to '<API_Key>'
+        Then 'Firebolt' platform responds with '<Method_Validation_Key>'
+        Given 'secondary 3rd party app' invokes the 'Firebolt' API to '<API_Key>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with 'null for getting stored value'
+
+        Examples:
+            | Scenario                                 | Set_API_Key                                           | API_Key                                                                 | Method_Validation_Key                                                | Clear_API_Key                                       |
+            | setForApp with device scope with 2 apps  | set secure data value for an app with scope device    | get stored value with scope as device and key as authTestTokenDevice    | expected value for authTestTokenDevice stored data in securestorage  | clear stored value with scope as device for an app  |
+
+
+    @SecureStorage @manageSDK
+    Scenario Outline: SecureStorage.removeForApp - Positive Scenario: <Scenario>
+        Given 3rd party 'certification' app is launched with 'secondary 3rd party app' appId
+        And 1st party app invokes the 'Firebolt' API to '<Clear_API_Key>'
+        And 'Firebolt' platform responds to '1st party app' for 'null for clearing stored value for an app'
+        When '3rd party app' invokes the 'Firebolt' API to '<Get_API_Key1>'
+        Then 'Firebolt' platform responds with 'null for getting stored value'
+        Given 'secondary 3rd party app' invokes the 'Firebolt' API to '<Get_API_Key1>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with 'null for getting stored value'
```

Comment:
Instead of setting it twice in line 158 and 160, is it possible to set both values in a single step ? does this api allow that

Reply:
we can only add one key and value at a time